sw/device/rom_exts/manifest.md

```diff
@@ -184,13 +190,15 @@ Notes:
     This is used when signing and validating the image. This happens in the
     Mask ROM, as well as during firmware update.
 
-1.  **Usage Constraints** This is a 32-bit enumeration value which denotes what
-    kinds of devices the ROM_EXT image may be used on.
+1.  **Usage Constraint** This is a 256-bit unsigned numeric value.
 
-    This is used by the Mask ROM to validate the image is running on the right
-    kind of hardware.
+    This allows the ROM_EXT author to constrain a ROM_EXT to a single device or
+    a group of devices, such that the ROM_EXT will not validate (and therefore
+    will not boot) on devices not in that group.
```

Comment:
Would it make sense to mention that usage constraint acts like a _byte_ mask where each bit determines whether the corresponding bytes will end up in the device usage value or not?

Reply:
Nope, explicitly it's down to the SKU to define how to interpret the usage constraint value to produce the device usage value (as long as both are predictable). It could be used as a mask, but it could also be used as a the input to a hash of some sort. 

The reason we do it this way is if later OT chips add more OTP or other values that might be included here, the function is still general enough that they can use the scheme, even if the exact computation performed is different.